.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,6 +18,9 @@ please refer to: <https://github.com/CICE-Consortium/About-Us/wiki/Resource-Inde
 - Does this PR create or have dependencies on Icepack or any other models?
     - [ ] Yes
     - [ ] No
+- Does this PR update the Icepack submodule?
+    - [ ] Yes
```

Comment:
```suggestion
    - [ ] Yes, does it point to a commit in Icepack's `main` branch ?
        - [ ] Yes
        - [ ] No
```

Reply:
I sort of disagree.  I think that's getting too deep in the weeds and folks that have erroneously included Icepack changes probably won't understand.  I think even the first question is unlikely to be answered correctly all the time when folks make a mistake.  I think the main point of the new question is to raise awareness that Icepack is relatively easy to change in CICE and to remind us reviewers to double-check.

Maybe what we really need is a review template that reviewers have to fill out.

- Is Icepack updated
- Is the documentation updated
- Does the documentation look OK
- Is testing adequate
...

Reply:
I think the confusing thing is that we always point to a "hash" of Icepack. So, this might be confusing mention main. Otherwise I do like this.

Reply:
Rather than asking this as a question, it could just be noted, e.g.
Does this PR update the Icepack submodule?  If so, it *must* point to a hash in Icepack's main branch.
yes/no

Reply:
I think that's a good idea !

Reply:
OK, I added an extra sentence.

Reply:
please change 'should' to 'must'

Reply:
@phil-blain, are you OK with the current state of changes and approve?